extensions/adobe/experience/engageable-profiles.example1.json

```diff
@@ -0,0 +1,20 @@
+{
+  "xdm:identityMap": {
+    "https://data.adobe.io/entities/namespace/4": [
+      {
+        "xdm:id": "92312748749128"
+      }
+    ],
+    "https://data.adobe.io/entities/namespace/10": [
+      {
+        "xdm:id": "2394509340-30453470347"
+      }
+    ]
+  },
+  "https://ns.adobe.com/experience/engageableProfiles": {
+    "xdm:engagement": {
+      "xdm:solution": "ajo",
```

Comment:
Shouldn't 'solution' be a list?? as a profile can engage with multiple solutions at a time.